src/test/java/janggi/board/point/PointTest.java

```diff
@@ -0,0 +1,58 @@
+package janggi.board.point;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class PointTest {
+
+    @DisplayName("X좌표와 Y좌표를 생성한다.")
+    @Test
+    void createTest() {
+        // given
+        int x = 1;
+        int y = 2;
+
+        // when & then
+        assertThatCode(() -> new Point(x, y))
+                .doesNotThrowAnyException();
+    }
+
+    @DisplayName("같은 수평선상에 있는지 확인한다.")
+    @Test
+    void horizontalTest() {
+        // given
+        Point point = new Point(0, 0);
+        Point otherPoint = new Point(2, 0);
+
+        // when & then
+        assertThat(point.isHorizontal(otherPoint))
+                .isTrue();
+    }
+
+    @DisplayName("같은 수직선상에 있는지 확인한다.")
+    @Test
+    void verticalTest() {
+        // given
+        Point point = new Point(0, 0);
+        Point otherPoint = new Point(0, 2);
+
+        // when & then
+        assertThat(point.isVertical(otherPoint))
+                .isTrue();
```

Comment:
```suggestion
        Point point = new Point(0, 0);
        Point otherPoint = new Point(0, 2);

        assertThat(point.isVertical(otherPoint)).isTrue();
```
만으로도 의미 전달이 명확한데 bdd에 얽매일 필요는 없어보입니다. 주석도 읽어야하는 1줄이라고 생각해보시면 좋겠네요.

Reply:
저는 `// given // when // then 주석`을 코드가 복잡하거나 길어질 때만 사용했던 적이 있습니다.
하지만 주석을 넣을지 말지를 고민하는 시간이 불필요하게 느껴졌습니다.

주석이 TDD를 연습하는 과정에서 테스트의 목표(무엇을 테스트하는가)를 명확히 하는 데 도움을 준다고 생각하여,
언제나 작성하는 방식으로 결정했고, 라이브 템플릿으로 등록하여 자동으로 적용하고 있습니다.

덕분에 일관된 구조로 TDD 연습을 할 수 있었습니다.

하지만 오리가 말한 것처럼, 주석도 읽어야 하는 코드의 일부인 것 같다는 의견에 공감되는 것 같아요. 👍 
그러나 현재 일관된 주석 구조가 제게 주는 이점이 크다고 생각합니다.

좋은 의견 주셔서 감사해요. 😄